pkg/services/accesscontrol/resourcepermissions/store_test.go

```diff
@@ -23,10 +22,10 @@ type setUserResourcePermissionTest struct {
 	resource          string
 	resourceID        string
 	resourceAttribute string
-	seeds             []types.SetResourcePermissionCommand
+	seeds             []SetResourcePermissionCommand
 }
 
-func TestAccessControlStore_SetUserResourcePermission(t *testing.T) {
+func TestStore_SetUserResourcePermission(t *testing.T) {
```

Comment:
These should probably be integration tests so that they run on all databases? `TestIntegrationStore…`